python/lib/cloudutils/configFileOps.py

```diff
@@ -59,39 +60,42 @@ def getEntry(self, name, separator="="):
             return ""
 
     def save(self):
-        fp = open(self.fileName, "r")
         newLines = []
-        for line  in fp.readlines():
-            matched = False
-            for entry in self.entries:
-                if entry.op == "add":
-                    if entry.separator == "=":
-                        matchString = "^\ *" + entry.name + ".*"
-                    elif entry.separator == " ":
-                        matchString = "^\ *" + entry.name + "\ *" + entry.value
-                else:
-                    if entry.separator == "=":
-                        matchString = "^\ *" + entry.name + "\ *=\ *" + entry.value
+        if os.path.exists(self.fileName) and os.path.isfile(self.fileName):
```

Comment:
So can we safely do without the libvirtd sysconfig file , @weizhouapache ?

Reply:
yes, 
it will then create a new file `/etc/sysconfig/libvirtd` with content `LIBVIRTD_ARGS=-l`